content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -452,12 +452,6 @@ of individual policies are not defined here.
 - {{< example file="security/podsecurity-baseline.yaml" >}}Baseline namespace{{< /example >}}
 - {{< example file="security/podsecurity-restricted.yaml" >}}Restricted namespace{{< /example >}}
 
-[**PodSecurityPolicy**](/docs/concepts/security/pod-security-policy/) (Deprecated)
-
-- {{< example file="policy/privileged-psp.yaml" >}}Privileged{{< /example >}}
-- {{< example file="policy/baseline-psp.yaml" >}}Baseline{{< /example >}}
-- {{< example file="policy/restricted-psp.yaml" >}}Restricted{{< /example >}}
```

Comment:
I think you can delete these example files. https://github.com/kubernetes/website/tree/main/content/en/examples/policy

Reply:
I'll leave this alone for now based on @sftim's comment